app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -97,11 +99,7 @@ export default class SingleAddon extends React.Component{
         </td>
         <td>
           {
-            app.appDetails && app.appDetails.uuid ?
-              app.appDetails.author :
-              app.appDetails && app.appDetails.developer ?
-                app.appDetails.developer.name :
-                app.appDetails.maintainers.map(maintainer => maintainer.name).join(', ')
+            (typeof maintainers === 'object') ?  maintainers.map(maintainer => maintainer.name).join(', ') : maintainers
```

Comment:
this type-checking isn't sufficient. Consider using _Array.isArray()_ instead, to assert that a variable is an array

Reply:
Thanks @segunolalive I have updated the PR